docs/helm-chart-checks.md

```diff
@@ -38,77 +39,77 @@ The following table lists the default set of checks with details including the n
+- GitHub profile to submit the chart to the [Red Hat OpenShift Helm Charts Repository](https://github.com/openshift-helm-charts).
+- Red Hat OpenShift Container Platform cluster.
 
 ### Procedure
-To perform the tasks related to Helm chart checks, run the following commands:
-
-- Runs all the available checks for the chart using a `uri`.
-
- ```ruby
- $ docker run -it --rm quay.io/redhat-certification/chart-verifier verify <chart-uri>
- ```
-- Runs all the checks available locally on your system for the chart, from the same directory as the chart.
-
- ```ruby
- $ docker run -v $(pwd):/charts --rm quay.io/redhat-certification/chart-verifier verify /charts/<chart>
- ```
-- Gets the list of options for the `verify` command.
-
- ```ruby
- $ docker run -it --rm quay.io/redhat-certification/chart-verifier verify help
- ```
- The output is similar to the following example:
- ```ruby
- $ docker run -it --rm quay.io/redhat-certification/chart-verifier verify help`
-
- Verifies a Helm chart by checking some of its characteristics
-
- Usage:
- chart-verifier verify <chart-uri> [flags]
-
- Flags:
-   -S, --chart-set strings          set values for the chart (can specify multiple or separate values with commas:     key1=val1,key2=val2)
-   -F, --chart-set-file strings     set values from respective files specified via the command line (can specify multiple or  separate values with commas: key1=path1,key2=path2)
-   -X, --chart-set-string strings   set STRING values for the chart (can specify multiple or separate values with commas: key1=val1,key2=val2)
-   -f, --chart-values strings       specify values in a YAML file or a URL (can specify multiple)
-   -x, --disable strings            all checks will be enabled except the informed ones
-   -e, --enable strings             only the informed checks will be enabled
-   -h, --help                       help for verify
-   -o, --output string              the output format: default, json or yaml
-   -s, --set strings                overrides a configuration, e.g: dummy.ok=false
-
- Global Flags:
-       --config string   config file (default is $HOME/.chart-verifier.yaml)
-- Runs a subset of the checks.
-
- ```ruby
- $ docker run -it --rm quay.io/redhat-certification/chart-verifier verify -e images-are-certified,helm-lint
- ```
-- Runs all the checks except a subset.
-
- ```ruby
- $ docker run -it --rm quay.io/redhat-certification/chart-verifier verify -x images-are-certified,helm-lint
- ```
-- Provides chart-override values.
-
- ```ruby
- $ docker run -it --rm quay.io/redhat-certification/chart-verifier verify -S default.port=8080 images-are-certified,helm-lint
- ```
-- Provides chart-override values in a file.
-
- ```ruby
- $ docker run -it --rm quay.io/redhat-certification/chart-verifier verify -F overrides.yaml images-are-certified,helm-lint
- ```
+
+- Run all the available checks for the chart using a `uri`:
```

Comment:
Awesome procedure! 

I'm used to seeing procedures ordered numerically, because the steps must be done in sequential order. Is there a reason why you used bullets instead of numbers?

Reply:
> Awesome procedure!
> 
> I'm used to seeing procedures ordered numerically, because the steps must be done in sequential order. Is there a reason why you used bullets instead of numbers?

@jc-berger Just that there is no specific order to execute the steps. They are pretty random. Hence the bulleted list. Thanks for checking :)